.github/workflows/pull-requests.yml

```diff
@@ -1,29 +1,48 @@
 name: GDevelop Issues automatic workflow
 on:
   pull_request:
-    types: [opened]
+    types: 
+      - opened
+      - reopened
+      - synchronize
 jobs:
   read-locales-metadata:
-    if: contains(github.event.pull_request.title, '[Auto PR] Update translations')
+    if: github.event.pull_request.title == '[Auto PR] Update translations'
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
       - name: Read and format locales metadata
+        env:
+          BASE: ${{ github.event.pull_request.base.sha }}
+          HEAD: ${{ github.event.pull_request.head.sha }}
         run: |
-          LANS=($(git diff HEAD^ HEAD --unified=5 newIDE/app/src/locales/LocalesMetadata.js | tail +6 | grep -E "^\s+\"languageName" | sed -E "s/^ *\"languageName\": \"//g" | sed -E "s/\",//g" | sed -E "s/ /_/g"))
-          ADDS=($(git diff HEAD^ HEAD --unified=0 newIDE/app/src/locales/LocalesMetadata.js | tail +6 | grep -E "^\+\s*\"translationRatio\"" | sed -E "s/^\+ *\"translationRatio\": //g"))
-          SUBS=($(git diff HEAD^ HEAD --unified=0 newIDE/app/src/locales/LocalesMetadata.js | tail +6 | grep -E "^\-\s*\"translationRatio\"" | sed -E "s/^\- *\"translationRatio\": //g"))
-          touch sumup.txt
+          LANS=($(git diff $BASE...$HEAD -- newIDE/app/src/locales/LocalesMetadata.js | grep -E "^\s+\"languageName" | sed -E "s/^ *\"languageName\": \"//g" | sed -E "s/\",//g" | sed -E "s/ /_/g"))
+          ADDS=($(git diff $BASE...$HEAD -- newIDE/app/src/locales/LocalesMetadata.js | grep -E "^\+\s*\"translationRatio\"" | sed -E "s/^\+ *\"translationRatio\": //g"))
+          SUBS=($(git diff $BASE...$HEAD -- newIDE/app/src/locales/LocalesMetadata.js | grep -E "^\-\s*\"translationRatio\"" | sed -E "s/^\- *\"translationRatio\": //g"))
+          touch sumup.md
+          echo "## Translation ratio changes" >> sumup.md
+          echo "" >> sumup.md
+          echo "| Language | Change |" >> sumup.md
+          echo "| --- | --- |" >> sumup.md
           for index in ${!ADDS[@]}; do
-            echo ${LANS[index]} | sed -E "s/_/ /g" >> sumup.txt
-            DELTA=$(bc <<< "scale=2;(${ADDS[index]}-${SUBS[index]})*100/1")
-            echo $DELTA % >> sumup.txt
+            LANGUAGE=${LANS[index]//_/ }
+            DELTA=$(echo "scale=6; (${ADDS[index]} - ${SUBS[index]})*100" | bc)
+            echo "| $LANGUAGE | $(printf "%.6f" $DELTA) % |" >> sumup.md
```

Comment:
You can use only 2 or 3 decimals I think. I had made this action in order to help for the check me make before merging the PR: Make sure there isn't a massive drop in translation ratios (that would mean there has been a problem somewhere on crowdin).

Reply:
Ahh, I see... How about we avoid commenting on the items that are showing as 0%?
![image](https://github.com/4ian/GDevelop/assets/88178296/40db7ef1-9c17-48e3-8ac9-4ab5101bccdf)

Reply:
That's a good idea if it's not too complex. Otherwise, it's already very nice!

Reply:
Done! You can see the appearance of the table after changes in the screenshot in the PR description!